examples/go.mod

```diff
@@ -0,0 +1,67 @@
+module github.com/go-swagger/go-swagger/examples
+
+go 1.21.4
```

Comment:
You might want align the go versions between root and examples

Reply:
yes sure. This is just a draft I have to make sure other things are not disturbed first (like go gettability etc)